src/api/auth.py

```diff
@@ -1,20 +1,29 @@
-from fastapi import Security, HTTPException, status, Request
-from fastapi.security.api_key import APIKeyHeader
+from fastapi import Depends, HTTPException
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+from supabase import create_client, Client
 import os
 import dotenv
 
 dotenv.load_dotenv()
 
-api_keys = []
+url: str = os.environ.get("SUPABASE_URL")
 
-api_keys.append(os.environ.get("API_KEY"))
-api_key_header = APIKeyHeader(name="access_token", auto_error=False)
+key: str = os.environ.get("SUPABASE_KEY")
```

Comment:
```suggestion
key: str = os.environ.get("SUPABASE_ANON_KEY")
```
I think that's what this is meant to represent? correct me if i'm wrong. it worked on my end when testing locally